salt/profiles/config/srv-profiles-clients.yaml

```diff
@@ -2,5 +2,5 @@
 {{ name }}:
   client_id: {{ details.client_id }}
   client_secret: {{ details.client_secret }}
-  redirect_uri: {{ details.redirect_uri }}
+  redirect_uris: {{ details.redirect_uris }}
```

Comment:
Does this work? It doesn't need the `json`/`yaml` filter?

Reply:
Still trying to provision locally due to the slow connection

Reply:
```
elife@dev--profiles:/srv/profiles$ cat clients.yaml 

some-app:
  client_id: foo
  client_secret: bar
  redirect_uris: ['https://example.com/check', 'https://testing.example.com/check']
```
I'll add the `yaml` filter.

Reply:
See also https://github.com/elifesciences/journal-formula/pull/44.